presto-docs/src/main/sphinx/release/release-334.rst

```diff
@@ -0,0 +1,87 @@
+=========================
+Release 333 (XX May 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect query results for certain queries involving comparisons of ``real`` and ``double`` types
+  when values include negative zero. (:issue:`3745`)
+* Fix failure when querying an empty table with late materialization enabled. (:issue:`3577`)
+* Fix failure when the inputs to ``UNNEST`` are repeated. (:issue:`3587`)
+* Fix failure when an aggregation is used in the arguments to :func:`format`. (:issue:`3829`)
+* Fix :func:`localtime` and :func:`current_time` for session zones with DST or with historical offset changes
+  in legacy (default) timestamp semantics. (:issue:`3846`, :issue:`3850`)
+* Improve performance of queries involving :func:`row_number`. (:issue:`3614`)
+* Improve performance of queries containing ``LIKE`` predicate. (:issue:`3618`)
+* Improve query performance when dynamic filtering is enabled. (:issue:`3632`)
+* Improve performance for queries that read fields from nested structures. (:issue:`2672`)
+* Add variant of :func:`random` function that produces a number between m and n. (:issue:`1848`)
+* Fix dynamic filter failures in complex spatial join queries. (:issue:`3694`)
+* Add :doc:`/connector/oracle`. (:issue:`1959`)
+* Add :doc:`/connector/pinot`. (:issue:`2028`)
+* Add support for modern HTTP forwarded headers. Processing of HTTP forwarded headers are now controlled by the
+  ``http-server.process-forwarded`` configuration, and the old ``http-server.authentication.allow-forwarded-https`` and
+  ``dispatcher.forwarded-header`` configurations are no longer supported. (:issue:`3714`)
+
+CLI Changes
+-----------
+
+* Improve rendering of elapsed time for short queries. (:issue:`3311`)
+
+Web UI Changes
+--------------
+
+* Add fixed, certificate, JWT and Kerberos to UI authentication. (:issue:`3433`)
+* Show join distribution type in Live Plan. (:issue:`1323`)
+
+JDBC Driver Changes
+-------------------
+
+* Improve performance of ``DatabaseMetaData#getColumns`` method in case when parameters
+  contain unescaped ``%`` or ``_``. (:issue:`1620`)
+
+Elasticsearch Connector Changes
+-------------------------------
+
+* Fix failure when executing ``SHOW CREATE TABLE``. (:issue:`3718`)
+* Improve performance for ``count(*)`` queries. (:issue:`3512`)
+* Add support for raw Elasticsearch queries. (:issue:`3735`)
+
+Hive Connector Changes
+----------------------
+
+* Fix creation of external tables using ``CREATE TABLE AS``. Previously such table was created
+  as managed and with default location. (:issue:`3755`)
```

Comment:
This was introduced recently (330). Do we wa want to mention when?

Reply:
What was introduced recently? The bug? Or the ability to create external tables?

Reply:
The bug _and_ ability to create external tables with data.